stream_alert/shared/__init__.py

```diff
@@ -1,6 +1,7 @@
 """Define some shared resources."""
 import logging
 import os
+import string
```

Comment:
I'm surprised pylint didn't complain about this! Just of curiosity, where did it come from?

Reply:
oh interesting - I think pylint won't complain about imports within `__init__` since it probable assumes you're doing package imports like `from . import *`